src/glog/logging.h.in

```diff
@@ -520,6 +565,11 @@ DECLARE_bool(stop_logging_if_full_disk);
 // specified by argv0 in log outputs.
 GOOGLE_GLOG_DLL_DECL void InitGoogleLogging(const char* argv0);
 
+#ifdef GLOG_ENABLE_CUSTOM_PREFIX
+GOOGLE_GLOG_DLL_DECL void InitGoogleLogging(const char* argv0,
+                                            PrefixAttacherFunction prefix_attacher);
```

Comment:
I'd rather call the callback `CustomPrefixCallback` or something along the lines.